src/gui/about.cpp

```diff
@@ -0,0 +1,56 @@
+//*****************************************************************
+/*
+  QJackTrip: Bringing a graphical user interface to JackTrip, a
+  system for high quality audio network performance over the
+  internet.
+
+  Copyright (c) 2020 Aaron Wyatt.
+  
+  This file is part of QJackTrip.
+
+  QJackTrip is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  QJackTrip is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with QJackTrip.  If not, see <https://www.gnu.org/licenses/>.
+*/
+//*****************************************************************
+
+#include "about.h"
+#include "ui_about.h"
+#include "../jacktrip_globals.h"
+
+const QString About::sBuildID = "2021050500";
+
+About::About(QWidget *parent) :
+    QDialog(parent),
+    m_ui(new Ui::About)
```

Comment:
Another fix would be to delete in the destructor. But as there's no memory management in the class necessary, I would prefer the other approach.

Reply:
There's no need to delete because m_ui is a scoped pointer that's assigned the new Ui::About object. This gets deleted automatically when m_ui goes out of scope.

Reply:
Oh, I missed that. Great :)